File: src/main/java/me/nylestroke/nylemod/block/custom/GrapeVineBlock.java
package me.nylestroke.nylemod.block.custom;

import me.nylestroke.nylemod.item.ModItems;
import net.minecraft.block.CropBlock;
import net.minecraft.item.ItemConvertible;

public class GrapeVineBlock extends CropBlock {

    public GrapeVineBlock(Settings settings) {
        super(settings);
    }

    @Override
    protected ItemConvertible getSeedsItem() {
        return ModItems.GRAPE_SEEDS;
    }
}

File: src/main/kotlin/me/nylestroke/nylemod/block/custom/GrapeVineBlock.kt
package me.nylestroke.nylemod.block.custom

import me.nylestroke.nylemod.item.ModItems
import net.minecraft.block.CropBlock
import net.minecraft.item.ItemConvertible

open class GrapeVineBlock(settings: Settings) : CropBlock(settings) {

    override fun getSeedsItem(): ItemConvertible {
        return ModItems.GRAPE_SEEDS
    }
}
